typespec-extension/readme.md

```diff
@@ -36,23 +36,87 @@ emit:
 options:
   "@azure-tools/typespec-java":
     emitter-output-dir: "{project-root}/azure-ai-language-authoring"
-    namespace: "com.azure.ai.language.authoring"
     service-name: "Authoring"
     generate-samples: true
     generate-tests: true
     partial-update: false
     api-version: "2023-11-01"
 ```
 
+### SDK Related Configuration in Emitter options `tspconfig.yaml`
+
+#### `api-version`
+
+By default, emitter generates code from the latest api-version in TypeSpec.
+
+In cases where a service needs to generate code from a previous api-version, set the value to that specific api-version.
+
+#### `service-name`
+
+This emitter option is for management-plane SDK.
+
+It is advised for service to set an appropriate service name. E.g. `service-name: Standby Pool`.
+
+The name will appear in documentations (e.g. "README.md") that describes the service and client. It would also affect the name of the entry class of the SDK.
+
+#### `partial-update`
+
+Default value is `false`.
+
+This emitter option is for data-plane SDK.
+
+In the case that the generated code is not good enough for the SDK, developer can choose to customize generated SDK via [Partial Update](https://github.com/Azure/azure-sdk-for-java/wiki/TypeSpec-Java-QuickStart#partial-update). Set the value to `true` to enable this feature.
+
+#### `generate-samples`
+
+Default value is `true`. Emitter generates code samples under `generated` package.
+
+For data-plane SDK, the generated samples are for reference. The motivation is to show how samples can be written. It is expected that developers write correct and concise samples outside of `generated` package.
+
+If there is customization of the generated SDK, the generated samples may not compile after customization. One can delete the `generated` package, set the value to `false` to disable the generated samples.
+
+#### `generate-tests`
+
+Default value is `true`. Emitter generates tests under `generated` package. The motivation is to show how samples can be written.
```

Comment:
```suggestion
Default value is `true`. Emitter generates tests under `generated` package.
```

Reply:
thanks. probably a paste triggered on the wrong position...